lnbits/helpers.py

```diff
@@ -166,3 +170,20 @@ def update_query(table_name: str, model: BaseModel, where: str = "WHERE id = ?")
     """
     query = ", ".join([f"{field} = ?" for field in model.dict().keys()])
     return f"UPDATE {table_name} SET {query} {where}"
+
+
+def is_valid_email_address(email: str) -> bool:
+    email_regex = r"\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,7}\b"
```

Comment:
This is too strict, there are new TLDs such as `.foundation` which do not match it - the TLD can be maximum [63 chars](https://stackoverflow.com/questions/9238640/how-long-can-a-tld-possibly-be).
Also I think we do not need boundaries since we are using the fullmatch.

```suggestion
    email_regex = r"[A-Za-z0-9\._%+-]+@[A-Za-z0-9\.-]+\.[A-Za-z]{2,63}"
```